tests/test_helpers/test_commands.py

```diff
@@ -258,6 +258,23 @@ def test_localcommands_demo(
     )
 
 
+@patch('invenio_cli.helpers.commands.subprocess')
+@patch('invenio_cli.helpers.commands.DockerHelper')
+@patch('invenio_cli.helpers.docker_helper.subprocess')
+def test_localcommands_destroy(
+        docker_helper_subprocess, patched_docker_helper, patched_subprocess,
+        fake_cli_config):
+    commands = LocalCommands(fake_cli_config)
+
+    commands.destroy()
+
+    patched_subprocess.run.assert_called_with(['pipenv', '--rm'], check=True)
+
+    docker_helper_subprocess.run.destroy_containers()
```

Comment:
With the changes above we can actually test now:

```suggestion
    docker_helper_subprocess.run.assert_called_with(['docker-compose', '--file', 'docker-compose.yml', 'down', '--volumes'])
```